pepys_import/core/store/data_store.py

```diff
@@ -623,6 +624,8 @@ def get_datafile(self, datafile_name=None, datafile_type=None):
         # Check for name match in Datafile and Synonym Tables
         if datafile_name:
             datafile = self.find_datafile(datafile_name=datafile_name)
+            # found object should be initialised because of _measurement variable
+            datafile.__init__()
             if datafile:
```

Comment:
@BarisSari - should we put the `__init__()` after the `if datafile:` test?
(I presume that test is present because `self.find_datafile()` could return `None`.

Reply:
You're right. We should put it after `if datafile`. I think we have tests where a datafile object is found an returned. That's why it doesn't fail even though `__init__()` is in the wrong place. I'm fixing it.

Reply:
Corrected:
https://github.com/debrief/pepys-import/pull/127/commits/4ef9f61f76745e6f842d3d03fa9724001234290c